server/src/main/java/org/elasticsearch/action/admin/indices/stats/TransportIndicesStatsAction.java

```diff
@@ -57,6 +57,11 @@ public TransportIndicesStatsAction(ClusterService clusterService, TransportServi
         this.indicesService = indicesService;
     }
 
+    @Override
+    protected boolean shouldIncludeDataStreams() {
```

Comment:
Note that this PR also adjusts the indices stats api in order to support resolving data streams. This is needed because rollover uses that api and we made that api already data stream ready. The reason why before we didn't have the change the indices stats api, is because rollover api overrides the default indices option when calling the indices stats api.